rehive/api/resources/admin_resources.py

```diff
@@ -173,9 +173,8 @@ def get_totals(self, **kwargs):
         response = self.get('totals/', **kwargs)
         return response
 
-    def patch(self, tx_code, status, **kwargs):
+    def patch(self, tx_code, **kwargs):
```

Comment:
@almabud2 in the original issue I asked whether this `patch` method is required? Did you look into this?

As far as I can tell, this resource inherits its patch method from here: https://github.com/rehive/rehive-python/blob/7b2c2f0348ad041e52b48bf85a6413704a5dba9f/rehive/api/resources/base_resources.py#L60

Which means you can probably just remove the `patch` entirely from `APIAdminTransactions` as the class it inherits from already performs the expected functionality.